.travis.yml

```diff
@@ -5,16 +5,51 @@ php:
   - 5.4
   - 5.5
   - 5.6
-  - 7.0
   - hhvm
-  
+  - nightly
+
+env:
+  - SYMFONY_VERSION=2.6.*
```

Comment:
-1 ofr this. The default case should not require ``symfony/symfony`` at all, to ensure that our requirements are right